### DISCUSSION TITLE
Remove numbers from table of contents ToC v.0.59.4

### DISCUSSION THREAD

**Discussion by Kani3D** [TriliumNext/Trilium#3873] [Q&A]
Hello,

first of all i want to say that triliumnotes is the tool i did not know i could live without.
Really massive thanks for the work on this project.
I'm going to start some discussions/questions because i came across some things.

Here is the first:

Is it possible to remove the numbers from the table of contents?
![image](https://user-images.githubusercontent.com/105086349/233950888-21178630-b1c7-4b9e-87e4-bb108dbd0af8.png)

Thanks in advance.

**Comment by trailjeep** [comment]
You can style pretty much everything with CSS code notes.

New note
type = css
owned attrs. = `#appCss`
content = 
```css
.toc ol {
    list-style: disc;
}
```
ref: [CSS list-style-type Property](https://www.w3schools.com/cssref/pr_list-style-type.php#Property%20Values)

**Reply by Kani3D** [reply]
Awesome!
Honestly, you thought me fishing right now.
Thanky you.

Is there also some syntax for "TABLE OF CONTENTS" ?

nvm found it:

#right-pane .card-header {
    background: inherit;
    padding: 6px 10px 3px 0;
    width: 99%;
    background-color: var(--button-background-color);
    border-color: var(--button-border-color);
    border-width: 0 0 1px 0;
    border-radius: 4px;
    border-style: solid;
    display: flex;
    justify-content: space-between;
    font-weight: bold;
    text-transform: uppercase;
    color: var(--muted-text-color) !important;
}

**Reply by Kani3D** [reply]
@trailjeep 
How can avoid my changes beeing overwritten?
![image](https://user-images.githubusercontent.com/105086349/234132322-f8aaaf1e-b8e3-4a03-8d1e-476be7927063.png)

**Reply by soulsands** [reply]
@Kani3D   Hi, you can add !important after a css rule value to prioritize it.  Check [this]( https://www.w3schools.com/css/css_important.asp  ) out to learn about the priority of css rules

**Reply by Kani3D** [reply]
> @Kani3D   Hi, you can add !important after a css rule value to prioritize it.  Check [this]( https://www.w3schools.com/css/css_important.asp  ) out to learn about the priority of css rules

@soulsands I have !important behind my color but it's also important in the default code.

**Reply by soulsands** [reply]
> > @Kani3D   Hi, you can add !important after a css rule value to prioritize it.  Check [this](https://www.w3schools.com/css/css_important.asp) out to learn about the priority of css rules
> 
> @soulsands I have !important behind my color but it's also important in the default code.

you can add a class selector in front of current one,  use `.component #right-pane .card-header`, it should work.
![image](https://user-images.githubusercontent.com/32272399/235080002-3e2af38a-dc77-4c86-9e97-ba93ab73a317.png)
![image](https://user-images.githubusercontent.com/32272399/235080072-5cdfa72c-cc19-4d15-8a24-69ce87d82bba.png)
